src/main/java/subway/service/LineService.java

```diff
@@ -0,0 +1,65 @@
+package subway.service;
+
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import subway.dto.*;
+import subway.jpa.Line;
+import subway.jpa.LineRepository;
+import subway.jpa.Station;
+import subway.jpa.StationRepository;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+@Transactional(readOnly = true)
+public class LineService {
+
+    private final LineRepository lineRepository;
+    private final StationRepository stationRepository;
+
+
+    public LineService(LineRepository lineRepository, StationRepository stationRepository) {
+        this.lineRepository = lineRepository;
+        this.stationRepository = stationRepository;
+    }
+
+    @Transactional
+    public LineDto saveLine(LineDto lineDto) {
+        Station upStation = stationRepository.findById(lineDto.getUpStationId())
+                .orElseThrow(() -> new IllegalArgumentException(String.format("상행역이 존재하지 않습니다. id:{}", lineDto.getUpStationId())));
```

Comment:
```suggestion
                .orElseThrow(() -> new IllegalArgumentException(String.format("상행역이 존재하지 않습니다. id:%s", lineDto.getUpStationId())));
```

와 같이 해야 의도하신 대로 동작할 것 같아요 😄